README.md

```diff
@@ -176,12 +181,3 @@ $ heroku drains:add https://logs.loggly.com/inputs/#{CUSTOMER_TOKEN}/tag/product
 This will tag all of the logs for the project with `production`, `elixir`, and
 the name of the project. This will allow you to more easily filter out the logs
 from other projects.
-
-## Dependencies
-
-- Erlang (latest)
-- Elixir (latest)
-- Hex
-- Rebar
-- Phoenix
-- Postgresql
```

Comment:
Earlier in the doc we specify that a list of dependencies is included but they have been taken out. We need to leave them in here or alter the above copy. My vote is to leave the list of dependencies so that people who puke when they hear docker will at least know what they have to install.